index.js

```diff
@@ -90,6 +90,9 @@ was tripped on ${new Date().toUTCString()}`);
                 if (err) {
                     // eslint-disable-next-line no-console
                     console.log(`Getting errors with ${JSON.stringify(args)}: ${err}`);
+                    if (err.message.indexOf('CircuitBreaker timeout') !== -1) {
```

Comment:
Somehow, I think it's better to check `err.status===undefined`

Reply:
You're right, fix it.

Reply:
It would be better to check `err.status` first to reduce the amount of calculation since you don't have to search for strings.